NDTensors/src/tensorstorage/default_storage.jl

```diff
@@ -3,12 +3,34 @@
 default_datatype(eltype::Type=default_eltype()) = Vector{eltype}
 default_eltype() = Float64
 
+## This is a system to determine if inds are blocked or unblocked used for default_storagetype
+@traitdef is_blocked_inds{indsT}
```

Comment:
Maybe even simpler and more general could be calling it just `is_blocked`. This would make it so that we don't need both `is_blocked_ind` and `is_blocked_inds`, and it may prove useful for other objects, like asking if a `Tensor` is blocked or not.